docs/Inputs.md

```diff
@@ -86,7 +102,7 @@ Then you can display a text input to edit the author first name as follows:
 ```jsx
 import { BooleanInput } from 'react-admin';
 
-<BooleanInput label="Commentable" source="commentable" />
+<BooleanInput label="Commentable" source="commentable" />;
```

Comment:
the addition of the final semicolon is wrong (and there are many of these in your commit)